src/OrleansProviders/Streams/Common/Monitors/MonitorAggregationDimensions.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Orleans.Runtime.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.Providers.Streams.Common
+{
+    public class MonitorAggregationDimensions
+    {
+        /// <summary>
```

Comment:
This is the file of Monitor Dimensions which used for aggregation , for example, user can aggregate on the dimension of host name, which is stored in NodeConfig